test/screenshot/mdc-button/custom/all.test.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
```

Comment:
I disagree. It's the only page that shows this icon positioning bug:

![image](https://user-images.githubusercontent.com/409245/37860075-afb34a18-2edb-11e8-8c76-b9cdb8abd51b.png)

Reply:
I might be inclined to agree with Lynn because some of the use cases this is testing seem invalid, which might also explain the "bug"...

For instance, you're applying the stroke mixins to buttons that don't include the `--stroked` class, which means that they have no `border-style` so there is no border and it therefore does not take up space, however the `stroke-width` mixin is still going to reduce padding under the assumption that there is a border applied.

TL;DR the button mixins only make sense when used in conjunction with the relevant modifier classes.

Reply:
Good point, Ken. I'll remove this file.